Resources/doc/configuration.rst

```diff
@@ -241,6 +241,7 @@ values:
                     depth: 512
                 json_deserialization:
                     options: 0 # json_decode options bitmask
+                    strict: true
```

Comment:
i think that the default value is `false`.

Reply:
```suggestion
                    strict: false
```